api-controllers/ServiceBrokerApiController.js

```diff
@@ -64,11 +64,17 @@ class ServiceBrokerApiController extends FabrikBaseController {
     const params = req.body;
     const planId = params.plan_id;
     const plan = catalog.getPlan(planId);
+    const context = _
+      .chain({})
+      .merge(req.params, req.body)
+      .set('plan', plan)
+      .value();
+    const dashboardUrl = this.getDashboardUrl(context);
 
     function done() {
       let statusCode = CONST.HTTP_STATUS_CODE.CREATED;
       const body = {
-        dashboard_url: ServiceBrokerApiController.getDashboardUrl(plan, req.params.instance_id)
+        dashboard_url: dashboardUrl
```

Comment:
`dashboard_url` is not mandatory response field. Can we please enhance it so that if a service does not want it, it does not send any dashboard url? We would need for k8s deployments to keep it optional.

Reply:
how do you propose we do this? Thats a new requirement, should there be another JIRA for it?

Reply:
I leave it to you. :)

I thought now that dashboard_url are not provided by SF and one can bring in their own, one can also ask how do they not send it. K8s based services would be those examples. We could conditionally check and send if needed may be.

Reply:
i was thinking of adding 'dashboard' as one of the supported features. If dashboard is supported, we check the template url and use it if its present else use SF default URL. But this would require changing product component of all services yml file and adding dashboard in them. Better to fix this additional skipping logic in another PR. 

Another way would be add an additinonal flag called 'dashboard_supported' at plan level whose default value is treated as true unless its defined and set to false. This way no changes in any existing services, but i prefer the first approach to this one as it goes well with the supported features logic.

Reply:
That's fine.